packages/card/index.js

```diff
@@ -1,93 +1,90 @@
-import { FabricWebComponent } from '../utils';
+import { html, css } from 'lit';
+import { FabricElement } from '../utils';
 import { card as c } from '@fabric-ds/component-classes';
```

Comment:
This package is no longer maintained. Should probably switch out for `@fabric-ds/css/component-classes`